### DISCUSSION TITLE
idea: add ability to let a random beatmap play during intro sequence for lazer

### DISCUSSION THREAD

**Discussion by Ronikusu** [ppy/osu#22871] [Ideas] [answered]
on osu! for windows, there is a setting in the main menu category that let a random beatmap play during the intro sequence. could this be put into lazer?

should go where the mouse is hovering:
![image](https://user-images.githubusercontent.com/124932615/225129216-49b860cf-87f7-4bc2-97b3-59bd2931d328.png)

**Comment by ItsShamed** [comment, marked as answer]
You see where your mouse is at? The setting toggle just above (“osu! music theme”) does exactly what you are describing! Try disabling it, and it will play the intro along with the audio of a random beatmap instead of the usual intro theme.

**Reply by Ronikusu** [reply]
i am blind
